rails/react-components/src/library/components/permission-forms-v2/students-tab/students-tab.tsx

```diff
@@ -69,33 +69,36 @@ export default function StudentsTab() {
           </thead>
           <tbody>
             {
-              teachers.map(teacher => (
-                <>
-                  <tr>
-                    <td>{teacher.name}</td>
-                    <td>{teacher.email}</td>
-                    <td>{teacher.login}</td>
-                    <td>
-                      <LinkButton onClick={() => handleViewClassesClick(teacher.id)} active={selectedTeacherId === teacher.id}>
-                        {
-                          selectedTeacherId === teacher.id ? "Hide Classes" : "View Classes"
-                        }
-                        {
-                          selectedTeacherId === teacher.id ? <i className="icon-caret-up" /> : <i className="icon-caret-down" />
-                        }
-                      </LinkButton>
-                    </td>
-                  </tr>
-                  {
-                    selectedTeacherId === teacher.id &&
-                    <tr className={css.expanded}>
-                      <td colSpan={4}>
-                        <ClassesTable teacherId={teacher.id} currentSelectedProject={currentSelectedProject} />
+              teachers.map(teacher => {
+                const active = selectedTeacherId === teacher.id;
+                return (
+                  <>
```

Comment:
I'm guessing this is generating a key warning.  You can use `<React.Fragment key={teacher.id}>` here to fix it.

Reply:
Thanks, fixed in recent commits.